Mariosyian/benchmarking-programming-languages
# Release #2 [v2.1.1]

## v2.1.1
- Flags 🎏 
  - Add -n|--name flag so the user can now choose a custom name for the output file.
- Documentation 📖 
  - Update documentation to include markdown parser.
- Benchmark script
  - Update elapsed time measurement to use both the Syrupy total time and the process run time.
  - Update the default 'RUNS' (set by the -r|--runs flag) to 100.
  - Update 'go test' function

## v2.1.0
- Test 🧑‍🎓 
  - Add haxe markdown parser tests (ensure they run successfully locally :D).
  - Add Sieve contains() function.
- Bugs 🐞 
  - Fix haxe markdown parser bug for image parsing.
- CircleCI ⚫⚪
  - Uncomment haxe from circleCI config.
  - Update circleCI config file to run 'haxelib setup' in current directory and install 'neko'.
- Miscellanous
  - Remove 'haxe_libs' directory as it would be best for each user to install them themselves (or use the Docker image) to avoid any conflicts.
